### ISSUE TITLE
Funds lost from way2collector

### ISSUE THREAD

**Issue by ThomasssArt** [kodadot/nft-gallery#10733]
One of the onboarded collectors lost his funds during minting because he got an error. Can you please check this. This would be the "most urgent" topic in my eyes...

Wallet address:
5H6ybEFCmNDbosZjx7cddcVKNCzu21SYegRmULf86Z49kSiB

![image](https://github.com/user-attachments/assets/baabc884-7463-4da6-84b6-efb2ec8eefcf)
![image](https://github.com/user-attachments/assets/52c5fb91-c3a8-4d6f-91f2-9f760ea91e87)

**Comment by vikiival** [comment]
He/She did not lost any funds

https://assethub-polkadot.subscan.io/account/5H6ybEFCmNDbosZjx7cddcVKNCzu21SYegRmULf86Z49kSiB

https://polkadot.subscan.io/account/163GjZWGd9V5FQaFukfdmmKUDpzYiJzgjBAFddeUee5fvx6h?tab=extrinsic


The second TX failed because he tried to mint NFT from H-emitters collection that is already minted out


https://assethub-polkadot.subscan.io/extrinsic/6814916-3


TL;DR
His funds are on AssetHub, no DOT were lost

**Comment by ThomasssArt** [comment]
> He/She did not lost any funds
> 
> https://assethub-polkadot.subscan.io/account/5H6ybEFCmNDbosZjx7cddcVKNCzu21SYegRmULf86Z49kSiB
> 
> https://polkadot.subscan.io/account/163GjZWGd9V5FQaFukfdmmKUDpzYiJzgjBAFddeUee5fvx6h?tab=extrinsic
> 
> The second TX failed because he tried to mint NFT from H-emitters collection that is already minted out
> 
> https://assethub-polkadot.subscan.io/extrinsic/6814916-3
> 
> TL;DR His funds are on AssetHub, no DOT were lost

Ok thanks a lot for the clarification and sorry for the missunderstanding!
